### PR TITLE
Device Triggers UI visitbility note added

### PR DESCRIPTION
**Pull request by spmvoss**
The Device Triggers configuration (through the UI) is only visible when Advanced Mode is enabled in the profile section.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @spmvoss, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20816).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/homekit.markdown [hunk 1]
```diff
@@ -429,7 +429,7 @@ The following integrations are currently supported:
 
 # Device Triggers
 
-Devices that support triggers can be added to the bridge by accessing options for the bridge in **{% my integrations title="Configuration >> Integrations" %}**.
+Devices that support triggers can be added to the bridge by accessing options for the bridge in **{% my integrations title="Configuration >> Integrations" %}**. Note that the Device Triggers configuration is only visible if Advanced Mode is enabled in your profile.
```
Comment (frenck) [inline source/_integrations/homekit.markdown]:
```suggestion
Devices that support triggers can be added to the bridge by accessing options for the bridge in **{% my integrations title="Configuration >> Integrations" %}**. To use this feature, Advanced Mode must be enabled in your user profile.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small suggestion on wording change.

**Review by frenck — APPROVED** [review]
Thanks, @spmvoss 👍

**Pull request merged**